extensions/grpc/deployment/src/test/java/io/quarkus/grpc/client/MutinyStubInjectionTest.java

```diff
@@ -45,7 +45,7 @@ public class MutinyStubInjectionTest {
     @Test
     public void test() {
         String neo = service.invoke("neo-mutiny");
-        assertThat(neo).startsWith("Hello neo-mutiny").doesNotContain("vert.x");
+        assertThat(neo).startsWith("Hello neo-mutiny").contains("vert.x");
```

Comment:
Without this failed with

```
java.lang.AssertionError: 

Expecting actual:
  "Hello neo-mutiny vert.x-eventloop-thread-0"
not to contain:
  "vert.x"

	at io.quarkus.grpc.client.MutinyStubInjectionTest.test(MutinyStubInjectionTest.java:48
```

With the change we should expect the client to run in the event loop. Maybe if this had been changed in the original PR it would've caught the bug.